README.md

```diff
@@ -300,7 +345,14 @@ compatible.both(Future.of('world')).value(console.log);
 ### Creating Futures
 
 #### Future
-##### `Future :: ((a -> (), b -> ()) -> Cancel) -> Future a b`
```

Comment:
It's worth mentioning that this change will partially break links. I don't think this is a problem, though.